api/v1/verticadb_webhook.go

```diff
@@ -1833,20 +1828,18 @@ func (v *VerticaDB) checkAnnotatedSubclustersInShutdownSandbox(old runtime.Objec
 	for sandboxName := range persistSandboxes {
```

Comment:
Why do we need the sandbox to be persistent?

Reply:
Because of this condition "oldSubcluster.Shutdown != newSubcluster.Shutdown"
The rule applies when a user tries to mess with an existing subcluster. If it is a new subcluster, it will be covered by other rules.

Reply:
That condition is for persistent subclusters, not for persistent sandboxes.

Reply:
If the sandbox is not persistent, it will be a new sandbox, For new sandboxes to be created, its shutdown field will not be true. We are only concerned about subclusters in persistent sandboxes.

Reply:
If we have a subcluster that isn't in a sandbox of old vdb, but we put it in the sandbox of new vdb, your code cannot cover that subcluster, right?

Reply:
Do we have a rule to prevent a "shutdown" subcluster to add to a sandbox in your PR?

Reply:
Correct myself. Rule 1 of https://jira.verticacorp.com/jira/browse/VER-97368 covers that scenario.

Reply:
> If we have a subcluster that isn't in a sandbox of old vdb, but we put it in the sandbox of new vdb, your code cannot cover that subcluster, right?

Right. That is the sandboxing scenario. It will be covered in https://jira.verticacorp.com/jira/browse/VER-97254